lang/LangSource/SC_LanguageConfig.hpp

```diff
@@ -65,6 +66,11 @@ class SC_LanguageConfig
 
 	bool forEachIncludedDirectory(bool (*)(const Path&)) const;
 
+	bool getExcludeDefaultPaths() const;
+	void setExcludeDefaultPaths(bool value);
+	void static processPathList(const char* nodeName, YAML::Node &doc, void (*func)(const Path&));
+	void static processBool(const char* nodeName, YAML::Node &doc, const std::function<void(bool)> &func);
```

Comment:
these don't need to be declared here, they could be static methods in the implementation file. that keeps stuff out of the public interface here.

Reply:
this would also allow you to remove the `#include <yaml-cpp/yaml.h>` from this header

Reply:
also, style in this codebase is overwhelmingly `static void` rather than `void static`.